src/Pos/Wallet/Web/Server/Lite.hs

```diff
@@ -86,3 +96,10 @@ convertHandler mws kd ws wsConn handler = do
   where
     excHandlers = [Catch.Handler catchServant]
     catchServant = throwError
+
+-- Stub implementations for lite wallet.
+instance Ether.MonadReader NodeDBs Production where
```

Comment:
all these stub implementations absolutely ugly and needed only for compilation lite-wallet, after refactoring of lite-wallet its must die